tests/repo/RepoLicense_test.cc

```diff
@@ -11,15 +11,24 @@ using namespace zypp;
 using std::cout;
 using std::endl;
 
-TestSetup test( Arch_x86_64 );
+
+static std::unique_ptr<TestSetup> testSetup;
```

Comment:
Basically right, but I'd prefer a solution that requires less syntactical changes in the existing code (no var rename, no var to ptr).
The `std::unique_ptr` can be used as [`_pimpl` for TestSetup](https://github.com/openSUSE/libzypp/blob/ma-draft/tests/lib/TestSetup.h). A testcase intro could then look like [this](https://github.com/openSUSE/libzypp/blob/ma-draft/tests/repo/RepoLicense_test.cc#L14). Usage of `test` in existing testcase would not need to be changed.

Reply:
Done